docs/modules/candid-guide/index.adoc

```diff
@@ -23,84 +27,99 @@ service counter : {
+- The `+get+` method reads the current value of a counter.
+- The `+subscribe+` method can be used to invoke another function, for example, to invoke a notification callback method each time the counter value changes.
 
-Every method has a sequence of argument and result types, and Internet-Computer specific annotations (such as `query`) are supported.
+As this example illustrates, every method has a sequence of argument and result types. 
+A method can also include annotations—like the `+query+` notation shown in this example—that are specific to the Internet Computer.
 
-Given this interface description, it is possible to interact with this service seamlessly from the command line, from a generic web interface, from Rust code etc.
+Given this simple interface description, it is possible for you to interact with this `+counter+` service directly from the command line or through a web-based front-end or programmatically from a Rust program or using through another programming or scripting language. 
 
-Furthermore, Candid supports the evolution of service interfaces: It precisely specifies which changes (such as adding new optional parameters) can be made, without breaking existing clients.
+In addition to interoperability, Candid supports the evolution of service interfaces by precisely specifying the changes that can be made without breaking existing clients. For example, you can safely add new optional parameters to a service without reconstructing the service or losing compatibility for existing clients.
 
 == Why create a new IDL?
 
-At first glance one may think that prior technologies, like JSON, XML, Protobuf etc., would suffice. However, Candid provides a certain combination of features that are not found in these technologies, and that are very valuable when developing on the Internet Computer.
-
-* JSON, XML etc. are merely data description languages. They describe how to map the values that we want to transfer to bytes or characters, but they do not describe the services as a whole.
-
-* Candid implementations map the Candid value directly to types and values of the host language; in Candid, programmers do not construct or deconstruct some abstract `Candid` value.
+At first glance, you might think that other technologies, such as JSON, XML, or Protobuf, would suffice. 
+However, Candid provides a unique combination of features that are not found in these other technologies.
+The features that make Candid particularly well-suited for developing applications for the Internet Computer include the following:
 
-* Candid defines what is a “safe upgrade” in a sound and compositional way.
+* Many interface description languages like JSON, XML, and Protobuf describe how to map individual values to bytes or characters, but they do not describe services as a whole. These languages focus on the data types you want to transfer instead of the methods that make use of those data types.
```

Comment:
The gist is that they aren't interface description languages (and not normally called that). They are merely data description languages. So I would reword that slightly.